sites/gc-page-feedback/includes/strings.liquid

```diff
@@ -0,0 +1,58 @@
+{%- comment -%}
+Set string variables
+{%- endcomment -%}
+
+{%- assign opuft_yes_en = "Yes" -%}
+{%- assign opuft_no_en = "No" -%}
+{%- assign opuft_problem1_label_en = "I can’t <strong>find</strong> the information" -%}
+{%- assign opuft_problem2_label_en = "The information is hard to <strong>understand</strong>" -%}
+{%- assign opuft_problem3_label_en = "There was an error or something <strong>didn’t work</strong>" -%}
+{%- assign opuft_problem4_label_en = "Other reason" -%}
+
+{%- assign opuft_yes_fr = "Oui" -%}
+{%- assign opuft_no_fr = "Non" -%}
+{%- assign opuft_problem1_label_fr = "Je ne peux pas <strong>trouver</strong> l’information" -%}
+{%- assign opuft_problem2_label_fr = "L’information est difficile à <strong>comprendre</strong>" -%}
+{%- assign opuft_problem3_label_fr = "Il y avait une erreur ou quelque chose <strong>ne fonctionnait pas</strong>" -%}
+{%- assign opuft_problem4_label_fr = "Autre raison" -%}
+
+{%- assign opuft_yes_value = opuft_yes_en | append: "-" | append: opuft_yes_fr -%}
+{%- assign opuft_no_value = opuft_no_en | strip_html | append: "-" | append: opuft_no_fr | strip_html -%}
+{%- assign opuft_problem1_value = opuft_problem1_label_en | strip_html | append: "-" | append: opuft_problem1_label_fr | strip_html -%}
+{%- assign opuft_problem2_value = opuft_problem2_label_en | strip_html | append: "-" | append: opuft_problem2_label_fr | strip_html -%}
+{%- assign opuft_problem3_value = opuft_problem3_label_en | strip_html | append: "-" | append: opuft_problem3_label_fr | strip_html -%}
+{%- assign opuft_problem4_value = opuft_problem4_label_en | strip_html | append: "-" | append: opuft_problem4_label_fr | strip_html -%}
+
+{%- if page.language == "en" -%}
+	{%- assign opuft_h3 = "Page feedback" -%}
+	{%- assign opuft_legend = "Did you find what you were looking for?" -%}
+	{%- assign opuft_yes = opuft_yes_en -%}
+	{%- assign opuft_no = opuft_no_en -%}
+	{%- assign opuft_why_note_nojs = "If not, tell us why below:" -%}
+	{%- assign opuft_why_note_js = "Tell us why below:" -%}
+	{%- assign opuft_no_legend = "What was wrong?" -%}
+	{%- assign opuft_problem1_label = opuft_problem1_label_en -%}
+	{%- assign opuft_problem2_label = opuft_problem2_label_en -%}
+	{%- assign opuft_problem3_label = opuft_problem3_label_en -%}
+	{%- assign opuft_problem4_label = opuft_problem4_label_en -%}
+	{%- assign opuft_problem6_label = "Please provide more details (maximum 300 characters)" -%}
+	{%- assign opuft_problem6_note = "You will not receive a reply. Telephone numbers and email addresses will be removed." -%}
+	{%- assign opuft_submit = "Submit" -%}
+	{%- assign opuft_submit_message = "Thank you for your feedback." -%}
+{%- else -%}
```

Comment:
This else statement can produce undesired side effects if a page language is not "en" nor "fr". The logic should explicitly run only if language is English or French.

Reply:
I don't think GCWeb currently officially-supports multilingual page templates, but I agree with your point :). I'll update it to something along the lines of ``elsif page.language == "fr"``.

Reply:
Updated :).